.tekton/build-pipeline.yaml

```diff
@@ -0,0 +1,581 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: build-pipeline
```

Comment:
If the project repo is not intented only for telco-core-rds I'd also be consistent in naming this pipeline to telco-core-rd-build-pipeline and adapt the naming in all accurences.